.github/workflows/lint.yml

```diff
@@ -0,0 +1,61 @@
+name: Lints
+
+on:
+  push:
+    # Prevent duplicate runs of this workflow on our own internal PRs.
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+        rust-toolchain: [nightly]
+
+    steps:
```

Comment:
I think it's worth adding a comment explaining why we have to do such an explosion of different clippy invocations here.

Reply:
I found the cargo package features RFC which provides a way to selectively enable feature flags in a workspace on some packages.

It's working really well. I was able to add `cargo clippy-legacy` and `cargo clippy-core` aliases.

https://doc.rust-lang.org/cargo/reference/unstable.html#package-features